plugins/rtorrent/rtom_allsessions_vol

```diff
@@ -80,48 +83,54 @@ my $src         = $ENV{"src"} || "";
+        } else {
+          $line = "<?xml version=\"1.0\" encoding=\"utf-8\"?><methodCall><methodName>d.multicall2</methodName><params><param><value><string></string></value></param><param><value><string>${_}</string></value></param><param><value><string>d.hash=</string></value></param></params></methodCall>";
+        }
+        my $llen = length $line;
+        my $header = "CONTENT_LENGTH\000${llen}\000SCGI\001\000";
+        my $hlen = length $header;
 
-	if ( ( defined $src ) && ( $src eq "socket" ) ) {
-	        for $socket (@sockets)
-	        {
-	                socket( SOCK, PF_UNIX, SOCK_STREAM, 0 ) or die;
-	                connect( SOCK, sockaddr_un( $socket ) ) or die $!;
-	                my $line = "${hlen}:${header},${line}";
-	                print SOCK $line;
-	                flush SOCK;
-	                while ( $line = <SOCK> ) {
-						if ( $line =~ /$pattern/ ) {
-							$num++;
-						}
-					}
-					close (SOCK);
-	        }        
-	} else {
-	        for $port (@ports)
-	        { 
-	                socket( SOCK, PF_INET, SOCK_STREAM, getprotobyname( "tcp" ) );
-	                connect( SOCK, sockaddr_in( $port, inet_aton( $ip ) ) );
-	                my $line = "${hlen}:${header},${line}";
-	                print SOCK $line;
-	                flush SOCK;
-	                while ( $line = <SOCK> ) {
-						if ( $line =~ /$pattern/ ) {
-							$num++;
-						}
-					}
-					close (SOCK);
-	        }
-	}
-	print "${_}.value ${num}\n";
+        if ( ( defined $src ) && ( $src eq "socket" ) ) {
+                for $socket (@sockets)
+                {
+                        socket( SOCK, PF_UNIX, SOCK_STREAM, 0 ) or die;
+                        connect( SOCK, sockaddr_un( $socket ) ) or die $!;
+                        my $line = "${hlen}:${header},${line}";
+                        print SOCK $line;
+                        flush SOCK;
+                        while ( $line = <SOCK> ) {
+                                if ( $line =~ /$pattern/ ) {
+                                                        $num++;
+                                                }
+                                        }
+                                        close (SOCK);
+                }
+        } else {
+                for $port (@ports)
+                {
+                        socket( SOCK, PF_INET, SOCK_STREAM, getprotobyname( "tcp" ) );
+                        connect( SOCK, sockaddr_in( $port, inet_aton( $ip ) ) );
+                        my $line = "${hlen}:${header},${line}";
+                        print SOCK $line;
+                        flush SOCK;
+                        while ( $line = <SOCK> ) {
+                                                if ( $line =~ /$pattern/ ) {
```

Comment:
Since you are changing the whitespace here (tabs to spaces), please feel free to fix the indentation in this block.
(the nesting levels look weird)